app/ntpd/README.md

```diff
@@ -24,74 +12,60 @@ that a local cluster of ntpd services stays in sync with an NTP pool.
-* ntpd stability (rabe.ntpdc.sysinfo[stability])
-* ntpd broadcastdelay (rabe.ntpdc.sysinfo[broadcastdelay])
-* ntpd authdelay (rabe.ntpdc.sysinfo[authdelay])
-
-#### Client
-
-The following Items are created for each discovered Server
-
-* NTP server {#SERVERNAME} candidate order (rabe.ntpd.server.candidate_order[{#SERVERNAME}])
-* NTP server {#SERVERNAME} configuration (vfs.file.regexp[/etc/ntp.conf,"^server.*{#SERVERNAME} (.*)",,,,\1])
+This template is part of [RaBe's Zabbix template and helpers
+collection](https://github.com/radiorabe/rabe-zabbix).
 
-#### Server
+## Template App ntpd Client active
+### Discovery
+
+#### Discovery Items
+* NTP server {#SERVERNAME} candidate order (`rabe.ntpd.server.candidate_order[{#SERVERNAME}]`)
+* NTP server {#SERVERNAME} configuration (`vfs.file.regexp[/etc/ntp.conf,"^server.*{#SERVERNAME} (.*)",,,,\1]`)
+#### Discovery Triggers
+* NTP server {#SERVERNAME} is not a valid candidate on {HOST.NAME} (`{Template App ntpd Client active:rabe.ntpd.server.candidate_order[{#SERVERNAME}].last()}=0`)
+
+If a configured server has a candidate order of 0 it is not considered as a valid time source and most likely has some issues.
+
+## Template App ntpd Common active
+
+### Items 
+* Memory usage ($5) of "$1" processes (`proc.mem[ntpd,ntp,,,rss]`)
+* Number of "$1" processes (`proc.num[ntpd,ntp]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[authdelay,single]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[broadcastdelay,single]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[jitter,single]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[precision]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[root dispersion,single]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[root distance,single]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[stability,single]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[stratum]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[system flags]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[system peer mode]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[system peer]`)
+## Template App ntpd Server active
 
 ### Triggers
 
-#### Common
+* High: No running ntpd processes on {HOST.NAME} (`{Template App ntpd Common active:proc.num[ntpd,ntp].max(#5)}<1`)
 
-* High: No running ntpd processes on {HOST.NAME}
+We expect ntpd to to run at all times
 
-* Warning: ntpd system peer mode is not client on {HOST.NAME}
-
-  Gets triggered when a system does not identify itself as in system peer mode "client".
-
-#### Client
-
-The following Triggers are created for each discovered Server
-
-* Warning: NTP server {#SERVERNAME} is not a valid candidate  on {HOST.NAME}
-
-  Gets triggered when a configured server is considered as invalid as indicated by it having a candidate order of 0.
-  
-#### Server
-
-### Discovery Rules
-
-* NTP servers (rabe.ntpd.server.discovery)
-
-### Macros
+* Warning: ntpd system peer mode is not client on {HOST.NAME} (`{Template App ntpd Common active:rabe.ntpdc.sysinfo[system peer mode].regexp(client)}<>0`)
 
+Out clients should always be in client peer mode. If they are not, chances are that they are not clients any more.
```

Comment:
I don't understand what an _out client_ is. Is this the correct term or simply a typo?